.github/workflows/build.yaml

```diff
@@ -36,6 +36,25 @@ jobs:
         if: always()
         with:
           check_name: 'Unit Test Report for Java ${{ matrix.java }}'
+  build-and-test-nflow-explorer-ng:
+    if: always()
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out
+        uses: actions/checkout@v3
+      - name: Set up Java
+        uses: actions/setup-java@v3
+        with:
+          java-version: 11
+          distribution: 'temurin'
+          cache: 'maven'
+      - name: Run build and unit tests
+        run: mvn -T1C --no-transfer-progress --color=always -Dmaven.javadoc.skip=true -Dmaven.source.skip=true -am -pl nflow-explorer-ng test
+      - name: Publish JUnit test report
+        uses: scacap/action-surefire-report@v1
+        if: always()
+        with:
+          check_name: 'Unit Test Report for nflow-explorer-ng'
```

Comment:
This is failing as there are no results found, should this be removed?

Reply:
Removed